src/DotVVM.Framework/ResourceManagement/LocalResourceUrlManager.cs

```diff
@@ -52,16 +79,28 @@ protected virtual string DecodeResourceName(string name)
         public ILocalResourceLocation FindResource(string url, IDotvvmRequestContext context, out string mimeType)
         {
             mimeType = null;
-            if (DotvvmRoutingMiddleware.FindMatchingRoute(new[] { resourceRoute }, context, out var parameters) == null) return null;
-            var name = DecodeResourceName((string)parameters["name"]);
-            var hash = (string)parameters["hash"];
+            if (DotvvmRoutingMiddleware.FindMatchingRoute(new[] { resourceRoute }, context, out var parameters) == null)
+            {
+                return null;
+            }
+
+            var name = DecodeResourceName((string)parameters[NameParameterName]);
+            var hash = (string)parameters[HashParameterName];
```

Comment:
Why are the lines above just reformatted?

Reply:
Most cleanup I can refrain myself from doing. Overlong early returns on a single line in a file I'm already editing I cannot ignore. I like seeing where the method returns clearly.